### PR TITLE
21634 Update NR request type code

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21634

*Description of changes:*
- app version = 7.2.13
- deleted local Name Request Types
- imported NR Request Action Codes from shared repo
- replaced old enum with new one
- copied Request Type switch statement from Create UI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/NameRequestInfo.vue [hunk 1]
```diff
@@ -124,10 +125,13 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
...
-    if (this.nameRequest.request_action_cd === NameRequestTypes.AMALGAMATION) {
-      return 'Amalgamation'
+    switch (this.nameRequest.request_action_cd) {
+      case NrRequestActionCodes.NEW_BUSINESS: return 'New Business'
+      case NrRequestActionCodes.RESTORE: return 'Restoration Request'
+      case NrRequestActionCodes.AMALGAMATE: return 'Amalgamation'
+      case NrRequestActionCodes.MOVE: return 'Continuation In'
     }
-    return 'New Business'
+    return '' // should never happen
```
Comment (severinbeauvais) [inline src/components/common/NameRequestInfo.vue:134]: Same code as Create UI.

File: src/enums/nameRequestTypes.ts
Comment (severinbeauvais) [inline src/enums/nameRequestTypes.ts:1]: Now using enum from shared repo.

File: src/components/common/NameRequestInfo.vue [hunk 2]
```diff
@@ -124,10 +125,13 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
 
   /** The request type. */
   get requestType (): string {
-    if (this.nameRequest.request_action_cd === NameRequestTypes.AMALGAMATION) {
-      return 'Amalgamation'
+    switch (this.nameRequest.request_action_cd) {
```
Comment (JazzarKarim) [inline src/components/common/NameRequestInfo.vue:128]: 👍 👍 👍

**Review by JazzarKarim — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-664-o8hej5g9.web.app

**Pull request merged**